packages/babel-traverse/src/scope/index.js

```diff
@@ -238,7 +271,7 @@ export default class Scope {
       .replace(/[0-9]+$/g, "");
 
     let uid;
-    let i = 0;
+    let i = 1;
```

Comment:
I don't quite get this change. Don't we now start with `_temp1`, although we should start with `_temp`

Reply:
Oh it is `> 1`, wonder if we should instead make `_generateUid()` do `>=1`. Although that might change all tests.

Guess we should do that separately.

Reply:
> Oh it is `> 1`, wonder if we should instead make `_generateUid()` do `>=1`. Although that might change all tests.
> 
> Guess we should do that separately.

Yep -- agreed on all points :-)

Reply:
(Here's how I put it [in the commit message](https://github.com/babel/babel/pull/9330/commits/4f2f5d26b863e11c0d58404a3ec9e5ef7e0b20da), a bit more verbosely:

```
We were trying the names

  _temp  _temp  _temp2 _temp3 ...

Skip the first one -- so it's just

  _temp  _temp2 _temp3 ...

(Alternatively, the hyper-optimized way would be like this:

  _temp  _temp0 _temp1 _temp2 ...

That's what C++ compilers do in name-mangling, to make it a little
less common to have to spend the extra character for `_temp10`.
But of course that breaks almost every fragile, output-dependent test
case in the codebase... 182/8641 tests in 32/134 suites, by my count.)
```
)